nilearn/input_data/masker_validation.py

```diff
@@ -0,0 +1,75 @@
+import warnings
+
+from .._utils.class_inspect import get_params
+from . import MultiNiftiMasker, NiftiMasker
+
+
+def check_embedded_nifti_masker(estimator, multi_subject=True):
+    """Base function for using a masker within a BaseEstimator class :
```

Comment:
This doesn't follow PEP 258 or whatever number the docstring PEP is. In particular you should have a short description on one line, then a newline and then a longer description.